<ui version="4.0" >
 <class>KexiDBConnectionWidget</class>
 <widget class="QWidget" name="KexiDBConnectionWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>414</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item rowspan="3" row="0" column="0" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QLabel" name="iconLabel" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>331</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>4</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" >
    <widget class="Q3ButtonGroup" name="locationBGrp" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Database Server</string>
     </property>
     <layout class="QGridLayout" >
      <item row="1" column="5" colspan="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>41</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="4" colspan="2" >
       <widget class="KIntNumInput" name="customPortEdit" >
        <property name="minValue" stdset="0" >
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2" >
       <widget class="QLabel" name="portLbl" >
        <property name="text" >
         <string>Port:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="5" >
       <widget class="KLineEdit" name="hostEdit" />
      </item>
      <item row="1" column="0" colspan="3" >
       <widget class="QRadioButton" name="localhostRBtn" >
        <property name="text" >
         <string>Local server</string>
        </property>
        <property name="shortcut" >
         <string/>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="6" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>90</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0" colspan="2" >
       <widget class="QLabel" name="hostLbl" >
        <property name="text" >
         <string>&amp;Hostname:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>hostEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2" >
       <widget class="QCheckBox" name="chkPortDefault" >
        <property name="text" >
         <string comment="port: default" >Default</string>
        </property>
        <property name="buttonGroupId" stdset="0" >
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3" colspan="2" >
       <widget class="QRadioButton" name="remotehostRBtn" >
        <property name="text" >
         <string>Remote server</string>
        </property>
        <property name="buttonGroupId" stdset="0" >
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="lblEngine" >
        <property name="focusPolicy" >
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="text" >
         <string>&amp;Engine:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring></cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="6" >
       <widget class="Q3Frame" name="frmEngine" >
        <property name="focusPolicy" >
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Plain</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="Q3GroupBox" name="authenticationGBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Authentication</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="userLbl" >
        <property name="text" >
         <string>&amp;Username:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>userEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="KLineEdit" name="userEdit" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="passwordLbl" >
        <property name="text" >
         <string>&amp;Password:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>passwordEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="KLineEdit" name="passwordEdit" >
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QCheckBox" name="chkSavePassword" >
        <property name="text" >
         <string>Save password in the shortcut file</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="Q3GroupBox" name="dbGroupBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Database</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="nameLabel" >
        <property name="text" >
         <string>&amp;Name:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>nameCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2" >
       <widget class="KLineEdit" name="titleEdit" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="titleLabel" >
        <property name="text" >
         <string>&amp;Title (optional):</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignTop</set>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>titleEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="KPushButton" name="btnLoadDBList" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Maximum" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize" >
         <size>
          <width>32</width>
          <height>32767</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="KComboBox" name="nameCombo" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable" >
         <bool>true</bool>
        </property>
        <property name="insertPolicy" >
         <enum>QComboBox::InsertAtBottom</enum>
        </property>
        <property name="autoCompletion" >
         <bool>false</bool>
        </property>
        <property name="trapReturnKey" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Q3Frame" name="frmBottom" >
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Plain</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lblEngine</tabstop>
  <tabstop>frmEngine</tabstop>
  <tabstop>localhostRBtn</tabstop>
  <tabstop>remotehostRBtn</tabstop>
  <tabstop>hostEdit</tabstop>
  <tabstop>chkPortDefault</tabstop>
  <tabstop>userEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>chkSavePassword</tabstop>
  <tabstop>nameCombo</tabstop>
  <tabstop>btnLoadDBList</tabstop>
  <tabstop>titleEdit</tabstop>
 </tabstops>
 <includes>
  <include location="local" >knuminput.h</include>
  <include location="local" >klineedit.h</include>
  <include location="local" >kpushbutton.h</include>
  <include location="local" >kcombobox.h</include>
  <include location="local" >klineedit.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
